plugins/chat/spec/system/channel_about_page_spec.rb

```diff
@@ -79,15 +80,16 @@
 
     it "can edit name" do
       chat_page.visit_channel_about(channel_1)
-      find(".edit-name-slug-btn").click
 
-      expect(find(".chat-channel-edit-name-slug-modal__name-input").value).to eq(channel_1.title)
+      edit_modal = chat_channel_about_page.open_edit_modal
+
+      expect(edit_modal).to have_name_input(channel_1.title)
 
       name = "A new name"
-      find(".chat-channel-edit-name-slug-modal__name-input").fill_in(with: name)
-      find(".create").click
 
-      expect(page).to have_content(name)
```

Comment:
This assertion too wide, the content could be anywhere on the page. It is better to be specific in system tests.

Reply:
Yes we know this now, many of these specs have been written at the very beginning of our system specs adventures.